src/zimlet/com_zimbra_about/com_zimbra_about.properties

```diff
@@ -16,5 +16,5 @@
 
 label = About Zimbra
 description = Displays Zimbra version
-copyright = Copyright � 2005-2011 VMware, Inc. VMware and Zimbra are registered trademarks or trademarks of VMware, Inc. 
+copyright = Copyright © 2005-2011 VMware, Inc. VMware and Zimbra are registered trademarks or trademarks of VMware, Inc. 
```

Comment:
I only fixed the encoding here because it messed up my script. It was the only file with wrong encoding. 
It's not translated in any language, so I don't think it matters. Changed it to Synacor, just in case.